KerbalStuff/blueprints/lists.py

```diff
@@ -1,21 +1,21 @@
 import json
-from typing import Tuple, List, Union
+from typing import Tuple, List, Union, Optional
 
 from flask import Blueprint, render_template, url_for, abort, redirect, request
 from flask_login import current_user
 from sqlalchemy import desc
 import werkzeug.wrappers
 
-from ..common import loginrequired, with_session
+from ..common import loginrequired, with_session, get_game_info, paginate_mods
 from ..database import db
 from ..objects import Mod, ModList, ModListItem, Game
 
 lists = Blueprint('lists', __name__, template_folder='../../templates/lists')
 
 
 def _get_mod_list(list_id: str) -> Tuple[ModList, Game, bool]:
-    mod_list = ModList.query.filter(ModList.id == list_id).first()
-    ga = Game.query.filter(Game.id == mod_list.game_id).first()
+    mod_list = ModList.query.get(list_id)
+    ga = Game.query.get(mod_list.game_id)
```

Comment:
This line throws a NullRef if there is no list with that id. It should probably be moved two lines down.